app/models/chargeback.rb

```diff
@@ -21,6 +21,7 @@ def self.build_results_for_report_chargeback(options)
     rate_cols = ChargebackRate.where(:default => true).flat_map do |rate|
       rate.chargeback_rate_details.map(&:metric).select { |metric| perf_cols.include?(metric.to_s) }
     end
+    rate_cols.map! { |x| (x == "v_derived_cpu_total_cores_used") ? "cpu_usage_rate_average" : x }
```

Comment:
@gtanzillo @zeari I am agree that for 'Used CPU Cores' is badly used column `cpu_usage_rate_average` (https://github.com/ManageIQ/manageiq/pull/8547/files#diff-589c40657cf088c0212b48394e5dce41R23) because it seems that for ' 'Used CPU Cores' is proper virtual column `v_derived_cpu_total_cores_used`  but also it seems virtual column cannot be used in active record select method (https://github.com/ManageIQ/manageiq/blob/master/app/models/chargeback.rb#L24 - it is failing for me when virtual column is used.

and thanks to that it seems that we are not supporting  virtual columns in chargeback algorithm

@zeari  Was this your intension ? Briefly said : calculate 'Used CPU Cores' based on 
`v_derived_cpu_total_cores_used` instead of unrelated metric `cpu_usage_rate_average` ?